### PR TITLE
Avoid query on independent variable refresh

### PR DESCRIPTION
**Pull request by hwandersman**
### Overview
Template variables in Grafana are normally stored surrounded by `${}`. This syntax notifies Grafana that it should update the panel when the variable is updated. This dependency works when a template variable is used in queries, but if this syntax is used in a transformation or panel display option, then the panel will be refreshed on any template variable update.

To avoid this we remove `${}` from the temp var name in the dashboard JSON and have the panel listen to the URL directly for updates.

Added logic in the Scene Viewer and Video Player panels to get updates from the dashboard URL and update the component accordingly.

* This PR uses a new version of the grafana-utils lib. This brings a requirement to configure AWS SDK JS V3 which uses a different interface for credentials.
* The Video Player component interface was also updated.

-----

### Test criteria:
Avoiding panel refreshes only works in Grafana v8.5+. Need to test in the latest (9.x) and <8.5 (AMG uses 8.4.7) for backwards compatibility.

Test that the old dashboard templates aren't broken with this change in both version, and test the new templates in both versions.

Use a dashboard with the following:
* Alarm List panel
* Selected alarm history panel (has a query with a template var connected to the alarm list and scene viewer)
* Scene Viewer panel
* Video Player panel

Clicking on: a tag in the scene, a tag in the hierarchy list of the scene, an alarm in the alarm list
Should do the following: 
* Selected alarm history panel query refreshes and loads data for the updated template variables
* In Grafana v8.5+: Scene viewer, alarm list, and video player should not update at all
* In Grafana v8.4.7: All panels may update and continue to work

Edge cases:
* Loading a page with variables already set in the URL
* Saving default template var values and reloading a page with a clean URL
* Using a template var not set by the alarm list or scene viewer on a panel - should not update

### CODE REVIEW

File: src/panels/video-player/VideoPlayer.tsx [hunk 1]
```diff
@@ -4,18 +4,52 @@ import 'aws-iot-twinmaker-grafana-utils/dist/index.css';
...
   const setVideoPlayer = useCallback(() => {
+    // Get variables from the URL
+    const queryParams = locationSearchToObject(search || '');
+    let entityId = queryParams[getUrlTempVarName(props.options.entityId)] as string;
+    let componentName = queryParams[getUrlTempVarName(props.options.componentName)] as string;
+    let kvsStreamName = queryParams[getUrlTempVarName(props.options.kvsStreamName)] as string;
+
+    // Double check default variable values if url is empty
+    if (!entityId) {
+      const entityIdVar = tempVarSyntax(props.options.entityId);
```
Comment (mukeshsahay) [inline src/panels/video-player/VideoPlayer.tsx:27]: The following 3 if blocks has same code - will be better to separate out in a function and avoid duplication.

**Review by mukeshsahay — CHANGES_REQUESTED** [review]

File: src/panels/video-player/VideoPlayer.tsx (hunk 1 above)
Comment (hwandersman) [reply src/panels/video-player/VideoPlayer.tsx:27]: Yes, will condense this in the next commit

File: src/panels/awsConfig.ts [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 export const getAwsConfig = (
   credentialsProvider: () => Promise<Credentials>,
+  credentialsProviderV3: CredentialProvider,
   updatedRegion?: string
 ): UxSDKAwsConfig => {
   const region = updatedRegion ? updatedRegion : defaultRegion;
   return {
     credentialsProvider,
+    credentialsProviderV3,
```
Comment (msshui-amzn) [inline src/panels/awsConfig.ts:15]: I don't see where this is used

Comment (hwandersman) [reply src/panels/awsConfig.ts:15]: This is used by the updated grafana-utils to call S3 in the SceneViewer and KVS in the VideoPlayer. This was a recent update that uses both AWS SDK V2 and V3.

**Review by mukeshsahay — APPROVED** [review]

**Comment by mukeshsahay** [comment]
UxSDK version updated to 1.0.952.0

**Pull request merged**